app/bin/tools/backfill_user_ids.dart

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:gcloud/db.dart';
+import 'package:uuid/uuid.dart';
+
+import 'package:pub_dartlang_org/account/models.dart';
+import 'package:pub_dartlang_org/frontend/models.dart';
+import 'package:pub_dartlang_org/frontend/service_utils.dart';
+
+final _uuid = Uuid();
+
+Future main(List<String> args) async {
```

Comment:
Can we give the script a `--help` argument... and can we write a nice multi-line string that explain what it does.
Something like:
> Usage: dart backfull_user_ids.dart
Ensures a `User` entity exists for each `email` used in `PackageVersion.uploaderEmail` and `Package.uploaderEmails`.
Ensures that `Package.uploaders` is a subset of `userId`s for emails in `Package.uploaderEmails`.
Ensures that `PackageVersion.uploader` is the `userId` of `PackageVersion.uploaderEmail`.